tests/common/encoding.h

```diff
@@ -0,0 +1,47 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Common utility functions used by the encoding-related unit tests.
+ */
+
+#ifndef TESTS_ENCODING_H
+#define TESTS_ENCODING_H
+
+#include <cstdio>
+#include <cstdlib>
+
+#include "src/common/encoding.h"
+
+#ifndef SKIP_RETURN_CODE
+	#define SKIP_RETURN_CODE 77
+#endif
+
+static const int kSkipTest = SKIP_RETURN_CODE;
```

Comment:
Why bother with the `kSkipTest` constant at this point when the `SKIP_RETURN_CODE` macro already exists?

Shouldn't this logic be in a common place anyway?